test/built-ins/Temporal/Calendar/prototype/fields/long-input.js

```diff
@@ -10,22 +10,33 @@ info: |
   ## 12.4.21 Temporal.Calendar.prototype.fields ( fields )
```

Comment:
(in lines 6–8) description is now out of date, should be updated to something like "Temporal.Calendar.prototype.fields will throw when its input iterable yields an invalid field"